test/import-test.js

```diff
@@ -1,246 +1,267 @@
 const fs = require('fs');
 const {expect} = require('chai');
 const {setLogger} = require('../index');
-const {id, pid, expectAndGetElement, expectAndGetEntry, expectValue, expectPrimitiveValue, expectChoiceValue, expectCardOne, expectChoiceOption, expectField, expectConcept, expectIdentifier, expectPrimitiveIdentifier, expectNoConstraints, importFixture, importFixtureFolder } = require('../test/import-helper');
+const {id, pid, expectAndGetElement, expectAndGetEntry, expectValue, expectPrimitiveValue, expectChoiceValue, expectCardOne, expectChoiceOption, expectField, expectConcept, expectIdentifier, expectPrimitiveIdentifier, expectNoConstraints, importFixture, importFixtureFolder, testCIMPL6Export } = require('../test/import-helper');
 const {Version, IncompleteValue, ValueSetConstraint, CodeConstraint, IncludesCodeConstraint, BooleanConstraint, TypeConstraint, CardConstraint, TBD, REQUIRED, EXTENSIBLE, PREFERRED, EXAMPLE} = require('shr-models');
 const err = require('shr-test-helpers/errors');
+const shrexpand = require('shr-expand');
+const expand = shrexpand.expand;
+const errorLogger = err.logger();
 
+shrexpand.setLogger(errorLogger)
 // Set the logger -- this is needed for detecting and checking errors
-setLogger(err.logger());
-const writeCIMPL6 = true;
+setLogger(errorLogger);
 
-describe('#importDataElement', () => {
+//------------------- Controlling the Tests -------------------------
+
+// PHASE 1: Run import tests (always true)
+const phase1 = true; 
+// PHASE 2: Expand and Export: Write the re-constituted files to dataElementExports
+const phase2 = true;
+// PHASE 3 (comment out if not desired): This re-runs the tests by importing the files that were exported to dataElementExports (only valid if files were exported in phase 1)
+const phase3 = true;
```

Comment:
It seems to me, if I read the code correctly, that even if this is set to `false`, the _import_ tests will still be run on the exported CIMPL6 code.  This flag will just govern whether or not it gets re-exported again?  Or am I reading the code wrong?

Reply:
Phase 2 is expand and export. It is possible to have phase2 = false and leave phase3 = true but that would be bad.

Reply:
I'm not sure I communicated my question well, but given that it's working the way you want it, I don't think I need to press it any more.